test/screenshot/spec/mdc-data-table/classes/baseline-checkbox.html

```diff
@@ -97,7 +97,7 @@
             </tr>
             <tr data-row-id="u2" class="mdc-data-table__row mdc-data-table__row--selected" aria-selected="true">
               <td class="mdc-data-table__cell mdc-data-table__cell--checkbox">
-                <div class="mdc-checkbox mdc-data-table__row-checkbox">
+                <div class="mdc-checkbox mdc-data-table__row-checkbox mdc-checkbox--selected">
```

Comment:
nit: this row is "Offline". Is the Online/Offline & checkbox supposed to be related?

Reply:
They're not related. Checkbox selects the table row.

See this for example: https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/06/04/16_18_48_988/spec/mdc-data-table/classes/baseline-checkbox.html?utm_source=golden_json